criu/pstree.c

```diff
@@ -241,6 +241,11 @@ int init_pstree_helper(struct pstree_item *ret)
 	return 0;
 }
 
+bool has_children(struct pstree_item *item)
```

Comment:
Could you also replace `!list_empty(&item->children)` with has_children() in `cr-dump.c`, `pstree.c` and `seize.c`?